.github/workflows/policy-ignored-lines.txt

```diff
@@ -0,0 +1,8 @@
+# Add one entry per line.  Each entry is a hash of a filename and source code content.
+
+5efe25bb437d2182f96ba6c89e850cae3287d8e08e4c3655c53870f87a067c96
```

Comment:
This is a bit of a nit and not sure if there's a great way to enforce this, but as a code reviewer, it's a little tough to tell which new line is being ignored. As a reviewer, I'd want to give the ignored line a quick look as a sanity check. Would it be too much overhead to force the author to include the unhashed filename and line (or line number) here?

Reply:
The line number is not considered in the hash as lines will move around.  I will force the filename to be added though.